packages/shared-docs/src/search.ts

```diff
@@ -84,14 +84,24 @@ export function createSearch(
     if (input.match(/^rand(om)?:/))
       return sampleArray(fuseCollection, limit)
 
+    // css attr
+    if (input.match(/^[a-zA-Z-]+:\s?[a-zA-Z0-9\s.#]+;?$/)) {
```

Comment:
I am not sure if this handling would be a bit too specific

Reply:
> I am not sure if this handling would be a bit too specific

I think you are right 😂 Maybe we can identify the attr by `/.*:.*/` or just simply use `includes(':')` ?